### PR TITLE
Fix jq error complaining about non-existing effective.config.json in install/uninstall logs

### PR DESCRIPTION
**Pull request by bhatpmk**
# Description

This PR fixes the jq error, complaining about non-existing effective.config.json when config.sh reads the value from CONFIG_JSON from function validate_config_json() before the step which generates "effective.config.json" and during undeploy.

As part of this change, the Jenkinsfile used by multi-cluster test is updated to archive the install and uninstall logs.

Fixes VZ-2564

# Checklist 

As the author of this PR, I have:

- [x] Checked that I included or updated copyright and license notices in all files that I altered
- [ ] Added or updated unit tests for any new functions I added
- [ ] Added or updated integration tests if appropriate
- [ ] Added or updated acceptance tests if appropriate

Code reviewer, please confirm this PR:

- [ ] Addressed the requirement and meets the acceptance criteria
- [ ] Does not introduce unrelated or spurious changes
- [ ] Does not introduce any unapproved dependency
- [ ] Makes sense and it easy to understand, and/or difficult areas of code are clearly documented so that they can be understood

### CODE REVIEW

File: platform-operator/scripts/install/config.sh [hunk 1]
```diff
@@ -39,9 +42,11 @@ function get_config_value() {
   if [ "$config_val" == "null" ]; then
     config_val=""
   fi
+
   # check if it is defined in the json files under install-overrides
-  if [ ! $(get_override_config_value "$jq_expr") == "" ]; then
-    config_val=$(get_override_config_value "$jq_expr")
+  local config_override_value=$(get_override_config_value "$jq_expr")
+  if [ ! "$config_override_value" == "" ]; then
```
Comment (desagar) [inline platform-operator/scripts/install/config.sh:48]: you can use != in shell i.e. `"$config_override_value" != ""`

**Review by michael-cico — APPROVED** [review]

File: platform-operator/scripts/install/config.sh (hunk 1 above)
Comment (bhatpmk) [reply platform-operator/scripts/install/config.sh:48]: Right, I just re-used the existing syntax.

File: platform-operator/scripts/install/config.sh [hunk 2]
```diff
@@ -159,7 +164,12 @@ function validate_config_json {
   set -o pipefail
   local jsonToValidate=$1
   echo "$jsonToValidate" | jq . > /dev/null || fail "Failed to read installation config file contents. Make sure it is valid JSON"
+}
 
+# Make sure the environmentName, dns entries and certificates are valid in CONFIG_JSON
```
Comment (bhatpmk) [inline platform-operator/scripts/install/config.sh:169]: We don't need a separate function validate_config_json_entries() now, will revert to the original function validate_config_json()

Comment (bhatpmk) [reply platform-operator/scripts/install/config.sh]: Reverted the earlier change to validate_config_json()

**Review by michael-cico — APPROVED** [review]

**Review by desagar — APPROVED** [review]

**Review by gilbode — APPROVED** [review]

**Pull request merged**